### PR TITLE
Add ETCD backup provider

### PR DESCRIPTION
**Pull request by majst01**

### CODE REVIEW

File: deploy/etcd-local.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTPS
+          initialDelaySeconds: 15
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 1
```
Comment (mwennrich) [inline deploy/etcd-local.yaml:69]: there is nothing listening on port 8080
```
default              0s          Warning   Unhealthy                 pod/etcd-main-0                               Readiness probe failed: Get https://10.244.0.6:8080/healthz: dial tcp 10.244.0.6:8080: connect: connection refused
```

File: deploy/etcd-s3.yaml [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTPS
+          initialDelaySeconds: 15
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 1
```
Comment (mwennrich) [inline deploy/etcd-s3.yaml:69]: same

Comment (majst01) [reply deploy/etcd-s3.yaml:69]: thanks for the hint, will do

File: deploy/etcd-local.yaml (hunk 1 above)
Comment (majst01) [reply deploy/etcd-local.yaml:70]: is /health according to documentation

**Review by Gerrit91 — APPROVED** [review]

**Review by mwennrich — APPROVED** [review]

**Pull request merged**